src/main/java/edu/usf/cutr/tba/model/TravelBehaviorRecord.java

```diff
@@ -25,31 +30,59 @@
             "Google Activity",
             "Google Activity Confidence",
             "Vehicle type",
-            "Activity Start Date and Time (UTC)",
-            "Origin location Date and Time (UTC)",
-            "Activity Start/Origin Time Diff (minutes)",
-            "Origin latitude",
-            "Origin longitude",
-            "Origin Horizontal Accuracy",
-            "Origin Location Provider",
-            "Activity Destination Date and Time (UTC)",
-            "Destination Location Date and Time (UTC)",
-            "Activity End/Destination Time Diff (minutes)",
-            "Destination latitude",
-            "Destination longitude",
-            "Destination Horizontal Accuracy",
-            "Destination Location Provider",
-            "Duration (minutes)",
-            "Origin-Destination Bird-Eye Distance (meters)",
+            "Activity Start Date and Time* (UTC)",
+            "Origin location Date and Time (*best) (UTC)",
+            "Activity Start/Origin Time Diff* (minutes)",
+            "Origin latitude (*best)",
+            "Origin longitude (*best)",
+            "Origin Horizontal Accuracy (*best)",
+            "Origin Location Provider (*best)",
+            "Activity Destination Date and Time* (UTC)",
+            "Destination Location Date and Time (*best) (UTC)",
+            "Activity End/Destination Time Diff* (minutes)",
+            "Destination latitude (*best)",
+            "Destination longitude (*best)",
+            "Destination Horizontal Accuracy (*best)",
+            "Destination Location Provider (*best)",
+            "Duration* (minutes)",
+            "Origin-Destination Bird-Eye Distance* (meters)",
             "Chain ID",
             "Chain Index",
             "Tour ID",
             "Tour Index",
             "Ignoring Battery Optimizations",
             "Talk Back Enabled",
-            "Power Save Mode Enabled"
+            "Power Save Mode Enabled",
+            "Origin fused Date and Time (UTC)",
+            "Origin fused latitude",
+            "Origin fused longitude",
+            "Origin fused Horizontal Accuracy",
```

Comment:
Could you add units for the horizontal accuracy headers?

`"Origin fused Horizontal Accuracy (meters)"`

Reply:
I made the change as required in the last push.